app/uk/gov/hmrc/tai/viewModels/TaxCodeViewModel.scala

```diff
@@ -49,6 +49,12 @@ object TaxCodeViewModel extends ViewModelHelper with TaxCodeDescriptor {
         appConfig)
     }
 
+    descriptionListViewModels.foreach { descriptionListViewModels =>
```

Comment:
remove this

Reply:
Not sure I can remove this, I can change the name to "descriptionListViewModel".

Reply:
This code is only printing key value pairs and nothing else. It just applies a side effect that would mess the output. If needed it would be preferable to use logs rather than prints but in this case printing this doesn't seem to add any benefit.

Reply:
agree